add jupytext to env and update lock files

JessicaS11 opened the pull request:
Hopefully will resolve failing checks in #12.

JessicaS11 committed:
add jupytext to env and update lock files

jomey commented:
Looks like it is still stumbling on the `jinja-markdown` requirement.
Did this build locally from scratch for you?

JessicaS11 commented:
I didn't try building locally - only updating the env/lock files.

jomey commented on conda/conda-osx-64.lock.yml:
```diff
@@ -374,8 +376,8 @@ dependencies:
   - jupyter-book=0.12.3=pyhd8ed1ab_0
   - pip:
-    - jinja-markdown === 1.210911 
+    - jinja-markdown === 1.210911 --hash=md5:None
```
This `--hash` change currently breaks the build.
I am looking into this on why this was added by the script

jomey replied:
Looks like [this step](https://github.com/snowex-hackweek/website2022/blob/d49ed4d8d8fc3f6bb82a8fba6408318ac9e2c58c/conda/lock-environment.sh#L35) in the `lock-environment.sh` file did succeed for you locally.

I will manually remove this hash which should fix this build

JessicaS11 replied:
I actually got an illegal flag note from sed on this line (sorry I didn't catch that in the output before):
```
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
rm: *.yml.bak: No such file or directory
```

jomey replied:
I assume you are on a Mac?

JessicaS11 replied:
Yup - that explains it!

jomey committed:
Conda - Lock files - Manually removing hash

The script to autogenerate seemed to have failed at the step
removing the empty hash declaration for the `jinja-markdown`
package.
